src/main/java/uk/gov/ons/ctp/response/casesvc/message/UploadObjectGCS.java

```diff
@@ -35,6 +33,18 @@ public boolean uploadObject(String filename, String bucket, byte[] data) {
     Boolean isSuccess = false;
     log.info("file_name: " + bucketFilename + " bucket: " + bucket + ", Uploading to GCS bucket");
     try {
+      // Override the wired Google Storage object (GCS) to use a local emulator
+      Storage storage =
+          StorageOptions.newBuilder()
+              .setCredentials(NoCredentials.getInstance())
+              .setProjectId("local")
+              .setHost("http://storage-emulator:9023")
+              .build()
+              .getService();
+      System.out.println(
+          "Storage Client initialized with storage-emulator endpoint "
+              + storage.getOptions().getHost());
+
```

Comment:
We will need to only create this `storage` object configured for the emulator when running integration tests or running in a local Docker network. This POC branch will only work outside of GCP as the above explicitly uses the emulator